wordclock/AddressableLedStrip.c

```diff
@@ -57,21 +58,20 @@ static void DisplayFrame(int frameIdx) {
 	TPixel* frame = _frames[frameIdx].buff;
 	uint8_t* flags = _frames[frameIdx].flags;
 
-	
 	for (int idx = 0; idx < _cols * _rows; idx++) {
 		if (flags[idx] & FLAG_IS_BG) {
 			frame[idx][RED_IDX] = _bgColor[RED_IDX];
 			frame[idx][GREEN_IDX] = _bgColor[GREEN_IDX];
 			frame[idx][BLUE_IDX] = _bgColor[BLUE_IDX];
 		}
-		if (g_settings.pixelType != PIXEL_TYPE_RGB) {
-			rgb2rgbw(frame[idx], g_settings.pixelType);
-		}
+		// For the RGBW leds the white LED is used to show shared value,
+		// The calculated RGBW value should not be stored in the frame, therefor make a copy
+		rgb2rgbw(_frameCopy[idx], frame[idx], g_settings.pixelType);
```

Comment:
Kun je de conversie niet in _writeFunction() doen?

Reply:
zat ik tussen te twijfelen